news/2015-08-10-Javascript-Support-In-Wildfly.adoc

```diff
@@ -134,7 +134,7 @@ $undertow
     .onGet("/rest/members",
         {headers: {"content-type": "application/json"}},
         ['db', function ($exchange, db) {
-            return DB.select("select * from members");
+            return DB.select("select * from member");
```

Comment:
the variable DB should be in lower case... like this: return db.select("select * from member");